docs/data/toolpad/building-ui/component-library.md

```diff
@@ -0,0 +1,23 @@
+# Component library
+
+<p class="description">Component library provides an easy way to construct your application UI</p>
+
+## Built in components
```

Comment:
What Grammarly suggests me: 

```suggestion
## Built-in components
```

Seems confirmed by https://ell.stackexchange.com/questions/16886/built-in-in-built-inbuilt-and-builtin-im-confused